birdhouse/components/weaver/docker-compose-extra.yml

```diff
@@ -30,6 +30,15 @@ services:
       - ./components/weaver/config/magpie/config.yml:/opt/local/src/magpie/config/permissions/weaver-permissions.cfg:ro
       - ./components/weaver/config/magpie/config.yml:/opt/local/src/magpie/config/providers/weaver-provider.cfg:ro
 
+  # extend twitcher with MagpieAdapter hooks employed for weaver proxied requests
+  twitcher:
+    environment:
+      MAGPIE_CONFIG_PATH: /opt/birdhouse/src/magpie/config.yml
```

Comment:
MagpieAdapter has always been installed with twitcher in the past, did not need this `MAGPIE_CONFIG_PATH` before?  

Now weaver is using this `MAGPIE_CONFIG_PATH`, if another service needs to also "customize" twitcher or the MagpieAdapter, it is stuck because `MAGPIE_CONFIG_PATH` already used?  Not saying we should fix this limitation now.  I just wanted to understand if this limitation exists.

Reply:
Yes, the rest is not used by Twitcher. Twitcher never loaded the config because it was not needed. It actually doesn't know anything about this config. It is only MapgieAdapter that knows what to do with it.

That's a good point about other configs. I will adjust it to use the directory approach to allow loading many configs per-service as done in Magpie.